pkg/repository/clone_mirror.go

```diff
@@ -299,33 +299,69 @@ func cloneComponents(repo *V1Repository,
 		compManifests[name] = newManifest
 	}
 
+	// Download TiUP binary
+	for _, goos := range options.OSs {
```

Comment:
We should probably share this code with the client code in v1repository, but we can refactor later.